### PR TITLE
bootc-switch: add page

### PR DESCRIPTION
**Pull request by renner0e**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/bootc-switch.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# bootc switch
+
+> Target a new container image reference to boot.
+> More information: <https://containers.github.io/bootc/man/bootc-switch.html>.
+
+- Change OS base to a new container image from a registry:
```
Comment (gutjuri) [inline pages/linux/bootc-switch.md:6]:
```suggestion
- Change the OS base to a new container image from a registry:
```

File: pages/linux/bootc-switch.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`sudo bootc switch {{image}}`
+
+- Change OS base to a new container image from the local image storage of the root user:
```
Comment (gutjuri) [inline pages/linux/bootc-switch.md:10]:
```suggestion
- Change the OS base to a new container image from the local image storage of the root user:
```

File: pages/linux/bootc-switch.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`sudo bootc switch --transport containers-storage {{image}}`
+
+- Change OS base to a new container image stored in a tarball:
```
Comment (gutjuri) [inline pages/linux/bootc-switch.md:14]:
```suggestion
- Change the OS base to a new container image stored in a tarball:
```

**Review by gutjuri — APPROVED** [review]
Thanks, LGTM after my suggestions :)

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**